### PR TITLE
zh-translation: /docs/ops/best-practices/deployment/index.md #1730

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1730](https://github.com/servicemesher/istio-official-translation/issues/1730)

### CODE REVIEW

File: content/zh/docs/ops/best-practices/deployment/index.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
-## Deploy fewer clusters
+## 部署较少的集群{#deploy-fewer-clusters}
 
-Deploy Istio across a small number of large clusters, rather than a large number
-of small clusters. Instead of adding clusters to your deployment, the best
-practice is to use [namespace tenancy](/zh/docs/ops/deployment/deployment-models/#namespace-tenancy)
-to manage large clusters. Following this approach, you can deploy Istio across
-one or two clusters per zone or region. You can then deploy a control plane on
-one cluster per region or zone for added reliability.
+应该在少量的大型群集（而不是大量的小型群集）中部署 Istio。最好的做法是使用[命名空间租赁](/zh/docs/ops/deployment/deployment-models/#namespace-tenancy)来管理大型集群，而不是将集群添加到部署中。按照这种方法，您可以在每个区域或区域中的一两个群集上部署 Istio。然后，您可以在每个区域或区域的一个群集上部署控制平面，以提高可靠性。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/best-practices/deployment/index.md]:
```suggestion
应该在少量的大型集群（而不是大量的小型集群）中部署 Istio。最好的做法是使用[命名空间租赁](/zh/docs/ops/deployment/deployment-models/#namespace-tenancy)来管理大型集群，而不是将集群添加到部署中。按照这种方法，您可以在每个区域或区域中的一两个集群上部署 Istio。然后，您可以在每个区域或区域的一个集群上部署控制平面，以提高可靠性。
```

File: content/zh/docs/ops/best-practices/deployment/index.md [hunk 2]
```diff
@@ -1,33 +1,22 @@
...
 
-## Deploy clusters near your users
+## 在靠近用户的地方部署集群{#deploy-clusters-near-your-users}
 
-Include clusters in your deployment across the globe for **geographic
-proximity to end-users**. Proximity helps your deployment have low latency.
+在全球部署群集，实现 **在地理位置上靠近终端用户**。这有助于降低 deployment 的延迟。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/best-practices/deployment/index.md]:
```suggestion
在全球部署集群，实现 **在地理位置上靠近终端用户**。这有助于降低 deployment 的延迟。
```

Comment (gorda) [reply content/zh/docs/ops/best-practices/deployment/index.md]: @yuxiaobo96 感谢细心 review，已修改

**Pull request merged**